experimental/unitsconversion/tests/data/baked/macros/units_info_v1.rs.data

```diff
@@ -0,0 +1,27 @@
0\0\x07\0\0\0\x08\0\0\0\t\0\0\0\x02\0\0S\0\xE0.y\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x05\0\0\0\x06\0\0\0\x07\0\0\0\x08\0\0\0\x01\0\0|\0\x01\x01\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x05\0\0\0\x07\0\0\0\x0B\0\0\0\x0C\0\0\0\x03\0\0S\0a\t\xE0\xF1\xEE\x07\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x05\0\0\0\x06\0\0\0\x07\0\0\0\x08\0\0\0\x01\0\0S\0\x05y\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x05\0\0\0\x06\0\0\0\x07\0\0\0\x08\0\0\0\x01\0\0S\0\x04y\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x14\0\0\0\x15\0\0\0\x16\0\0\0\x17\0\0\0\x03\0\0|\0\x02\0\0\x02\0\xFF\x03\0:\0\xFE\0\0S\0\x01\x01\0\x01\0\0\0\x05\0\0\0\0\0\0\0\n\0\0\0\x0B\0\0\0\x0C\0\0\0\r\0\0\0\x02\0\0S\0\xFE\0\0|\0\x01\x01\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x05\0\0\0\x0B\0\0\0\x10\0\0\0\x11\0\0\0\x01\x03\0:\09\xA9V]\x17\x08\0\xD0\xFD\xEE\x8D\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x0F\0\0\0\x1B\0\0\0\x1C\0\0\0\x1D\0\0\0\x01\x03\0:\0\x02\0\0S\0\xFD\0\0|\0\0\0\0\xF2\xED\xD4\xD4\x94\r\xA5<\x01\x01\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x05\0\0\0\x12\0\0\0\x13\0\0\0\x14\0\0\0\x01\x03\0:\0\0\0\x006?\x15\xD0\x1C(\xD5\x17\x19\x19\x01\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x05\0\0\0\t\0\0\0\n\0\0\0\x0B\0\0\0\x01\0\0S\0 \x8Aw)\x01\0\x01\0\0\x01\x05\0\0\0\0\0\0\0\x05\0\0\0\x0C\0\0\0\x14\0\0\0\x15\0\0\0\x02\0\0v\0\0i\xFE\r#>\x0FIf\x17\xFD0\xC2Y\x02\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x05\0\0\0\t\0\0\0\r\0\0\0\x0E\0\0\0\x01\x03\0:\0\xA3\xE3\xEC\x12\x80\xF0\xFA\x02\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x05\0\0\0\x06\0\0\0\x08\0\0\0\t\0\0\0\x01\0\0S\0\x02]\x02\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x05\0\0\0\t\0\0\0\x0F\0\0\0\x10\0\0\0\x03\0\0S\0\x99\x194\x1C\0\0\xA2\x94\x1A\x1D\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x05\0\0\0\t\0\0\0\x0F\0\0\0\x10\0\0\0\x03\0\0S\x003\xB3f\t\0\0\xA2\x94\x1A\x1D\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x0F\0\0\0\x10\0\0\0\x11\0\0\0\x12\0\0\0\x01\x03\0:\0\xFE\0\0|\0\xFF\0\0\x02\0\x01\x01\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x0F\0\0\0\x13\0\0\0\x14\0\0\0\x15\0\0\0\x01\x03\0:\0\x02\0\0S\0\xFE\0\0|\0\xD0\x80I\x06\x01\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x05\0\0\0\x07\0\0\0\n\0\0\0\x0B\0\0\0\x03\0\0S\0a\t\xEC\x07\x02\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x05\0\0\0\t\0\0\0\x0B\0\0\0\x0C\0\0\0\x01\x03\0:\0\x85 \xB4\x02P\xC3\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x05\0\0\0\x07\0\0\0\x08\0\0\0\t\0\0\0\x01\x03\0:\0\xE8\x03\x01\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x14\0\0\0\x15\0\0\0\x16\0\0\0\x17\0\0\0\x01\x03\0:\0\x02\0\0S\0\xFD\0\0|\0\xFF\0\0\x02\0\x01\x01\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x0F\0\0\0\x10\0\0\0\x11\0\0\0\x12\0\0\0\x01\x03\0:\0\x02\0\0S\0\xFD\0\0|\0\x01\x01\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x14\0\0\0\x15\0\0\0\x16\0\0\0\x17\0\0\0\x01\x03\0:\0\x02\0\0S\0\xFE\0\0|\0\xFF\0\0\x02\0\x01\x01\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x05\0\0\0\x08\0\0\0\t\0\0\0\n\0\0\0\x01\0\0|\0\x80:\t\x01\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x05\0\0\0\x08\0\0\0\t\0\0\0\n\0\0\0\x01\0\0|\0\x80:\t\x01\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x05\0\0\0\x07\0\0\0\t\0\0\0\n\0\0\0\x01\0\0S\0w\x04\xE2\x04\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x05\0\0\0\x06\0\0\0\x07\0\0\0\x08\0\0\0\x01\0\0\x96\0\x01\x01\0\x01\0\0\0\x05\0\0\0\0\0\0\0\x05\0\0\0\x06\0\0\0\x07\0\0\0\x08\0\0\0\x01\0\0\x96\0\x01\x01\0\x01") } };
```

Comment:
@robertbastian : how to read this in `experimental/unitsconversion/tests/units_test.rs`

Reply:
`icu_unitsconversion::provider::Baked::SINGLETON_UNITS_INFO_V1`

Reply:
Thx :)